### PR TITLE
don't demand to have a command

### PR DESCRIPTION
**Pull request by christian-bromann**
fixed regression with last release where a command was required

### CODE REVIEW

File: packages/wdio-cli/src/index.js [hunk 1]
```diff
@@ -5,25 +5,35 @@ import path from 'path'
...
      * if so, we assume the user ran `wdio path/to/wdio.conf.js` and we execute `wdio run` command
      */
     const params = { ...argv.argv }
-    const wdioConfPath = path.join(process.cwd(), params._[0])
-    const wdioConf = fs.existsSync(wdioConfPath)
-
-    if (wdioConf) {
-        return handler({ configPath: wdioConfPath, ...params })
+    if (!params._.find((param) => SUPPORTED_COMMANDS.includes(param))) {
+        params.configPath = path.join(process.cwd(), params._[0] || DEFAULT_CONFIG_FILENAME)
```
Comment (diachedelic) [inline packages/wdio-cli/src/index.js]: @christian-bromann this `path.join` appears to break support for absolute config paths, are they no longer supported?

Comment (mgrybyk) [reply packages/wdio-cli/src/index.js]: @diachedelic please raise an issue 1

Comment (diachedelic) [reply packages/wdio-cli/src/index.js]: See #4585

Comment (baruchvlz) [reply packages/wdio-cli/src/index.js]: I am working on changing this file in [this](https://github.com/webdriverio/webdriverio/pull/4570) pull request. I'll take a look at #4585 as well.

**Pull request merged**